docs/core/install/remove-runtime-sdk-versions.md

```diff
@@ -55,55 +65,68 @@ The package name for the .NET SDK installation for most package managers is `dot
+::: zone-end
 
-```bash
-version="1.0.1"
-sudo rm -rf /usr/share/dotnet/sdk/$version
-sudo rm -rf /usr/share/dotnet/shared/Microsoft.NETCore.App/$version
-sudo rm -rf /usr/share/dotnet/shared/Microsoft.AspNetCore.All/$version
-sudo rm -rf /usr/share/dotnet/shared/Microsoft.AspNetCore.App/$version
-sudo rm -rf /usr/share/dotnet/host/fxr/$version
-```
+::: zone pivot="os-linux, os-macos"
+
+### Scripted or manual
+
+If you installed .NET using the [dotnet-install script](linux-scripted-manual.md#scripted-install), or by [extracting a tarball](linux-scripted-manual.md#manual-install), you must remove .NET using the manual method.
 
 > [!IMPORTANT]
-> The version folders may not match the "version" you're uninstalling. The individual runtimes and SDKs that are installed with a single .NET release may have different versions. For example, you may have installed ASP.NET Core 5 Runtime, which installed the 5.0.2 ASP.NET Core runtime and the 5.0.8 .NET runtime. Each has a different versioned folder. For more information, see [Overview of how .NET is versioned](../versions/index.md).
+> This information is written for Linux, but also applies to macOS. On macOS, the root directory is most likely `/usr/local/share/dotnet/`. Substitute the directories accordingly.
 
-The parent directories for the SDK and runtime are listed in the output from the `dotnet --list-sdks` and `dotnet --list-runtimes` command, as shown in the earlier table.
+When you manually install .NET, it's generally installed to the `/usr/share/dotnet/` or `/usr/lib/dotnet/` directory. The SDK, runtime, and .NET host, are installed into separate sub directories. These "component" directories contain a directory for each version of .NET. By removing the versioned directories, you remove that version of .NET from your system. These directories may vary depending on your Linux distribution.
 
-::: zone-end
+There are three commands you can use to discover where .NET is installed: `dotnet --list-sdks` for SDKs, `dotnet --list-runtimes` for runtimes, and `dotnet --info` for everything. These commands don't list the .NET host. To determine which hosts are installed, check the `/usr/share/dotnet/host/fxr/` directory. The following list represents the directories of a specific version of .NET, where the `$version` variable represents the version of the .NET:
+
+- **SDK**:
+
+  `/usr/share/dotnet/sdk/$version/`
+
+- **Runtime**:
 
-::: zone pivot="os-macos"
+  The runtime is based on specific .NET product runtimes, such as ASP.NET Core or NETCore (the .NET runtime specifically). These are installed to the `/usr/share/dotnet/shared/$product/$version` directory, where `$product` is the product runtime. For example, you may see the following directories:
```

Comment:
Are the spellings of ASP.NET Core and NETCore right / consistent here?

Reply:
Thank you for spotting that.